lib/osv/ecosystems.py

```diff
@@ -381,3 +399,9 @@ def get(name: str) -> Ecosystem:
     return Debian(name.split(':')[1])
   else:
     return _ecosystems.get(name)
+
+
+def set_cache(redis_host: str, redis_port: str):
```

Comment:
Let's make this a bit more extensible rather than tying this too much to Redis. This also makes testing easier. 

We can create something like the following:

```python
class Cache:
  """Cache interface."""
  
  def get(key):
    raise NotImplementedError

  def set(key, value, expiry):
    raise NotImplementedError


class RedisCache(Cache):
  """Redis cache implementation."""

  def __init__(self, redis_host, redis_port):
   ....
  
  def get(...)

# If we need to in another PR. 
class InMemoryCache(Cache):
  ...
```


Then we can have set_cache take in instances of `Cache`.